client/jetpack-onboarding/steps/summary-todo-links.jsx

```diff
@@ -0,0 +1,58 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getEditorNewPostPath } from 'state/ui/editor/selectors';
+
+const TodoLinks = ( { steps } ) =>
+	map( steps, ( { label, url }, stepName ) => (
+		<div key={ stepName } className="steps__summary-entry todo">
+			<a href={ url }>{ label }</a>
+		</div>
+	) );
+
+export const TodoLinksConnected = connect( ( state, { siteId, siteSlug, translate } ) => ( {
+	steps: {
+		THEME: {
+			label: translate( 'Choose a Theme' ),
+			url: '/themes/' + siteSlug,
+		},
+		PAGES: {
+			label: translate( 'Add additional pages' ),
+			url: getEditorNewPostPath( state, siteId, 'page' ),
+		},
+		BLOG: {
+			label: translate( 'Write your first blog post' ),
+			url: getEditorNewPostPath( state, siteId, 'post' ),
+		},
+	},
+} ) )( TodoLinks );
+
+export const TodoLinksUnconnected = connect( ( state, { siteUrl, translate } ) => ( {
+	steps: {
```

Comment:
It seems that we don't need this layer of depth - we could be passing the steps directly as a parameter to `TodoLinks` - `( steps )` vs `( { steps } )`.

Reply:
04714117147d5ddf8a0d23946c9d1c91ac3ce59b